nservicebus/messaging/delayed-delivery.md

```diff
@@ -14,11 +14,15 @@ The message doesn't have to be dispatched immediately after sending, it can be d
 NOTE: Only send operations can be deferred. Publish and reply operations cannot be deferred.
 
 ## Handling current message later
-To handle the current message at a later time `bus.HandleCurrentMessageLater()` method can be used.  This was the initial way of *deferring* a message. This would put the message at the end of the queue and the message eventually will be pick up again once other messages are processed. The problem with this approach is that if there are no other messages in the queue (or that many of them) the message will pop right back up so the intended deferring might not happen. 
+To handle the current message at a later time `bus.HandleCurrentMessageLater()` method can be used.  This was the initial way of *deferring* a message. This would create a copy of the message that has the same identifier, header and body, and then put the new message at the end of the queue. The message eventually will be pick up again once other messages in the queue are processed. 
```

Comment:
I guess it's my OCD to have "snippets for all public" api's? 

That said we can deal with that separately if needed if @Particular/docs-maintainers is happy to pull this with no snippets